integration_tests/dbt_project.yml

```diff
@@ -19,10 +19,11 @@ vars:
   # shopify_gql_using_fulfillment_tracking_info: true # @docs-include
   # shopify_gql_using_product_variant_media: true # @docs-include
   
-  shopify_schema: combo_integration_tests_12
-  shopify_api_override: rest
+  # Only run both shopify schemas when regenerating docs. For integration tests, use the shopify_api variable to run one at a time
+  # shopify_api_override: rest # @docs-include
 
-  shopify_source:
+  shopify:
+    shopify_schema: combo_integration_tests_12
```

Comment:
Can we reset this to the non numeric increased schema. For example maybe something like `holistic_integration_tests`